Fail on invalid available-at when generating Gradle Module Metadata (#15905)
gradle/gradle: subprojects/docs/src/docs/design/gradle-module-metadata-latest-specification.md

This value, nested in `variants`, must contain an object with the following values:

> - `url`: The location of the metadata file that describes the variant. A string. In version 1.0 and 1.1, this must be a path relative to the module.

Reviewer: ```suggestion
- `url`: The location of the metadata file that describes the variant. A string. Since version 1.0, this must be a path relative to the module.
```

To have this future proof should there be a next version of the spec.

Author: I prefer being explicit and I do not think we have to do a new version for the changes reported here.
